resources/lib/npoapihelpers.py

```diff
@@ -3,6 +3,8 @@
 from resources.lib.jsonhelper import ToJsonObject
 import sys
 
+import xbmc
```

Comment:
Deze import is hier niet nodig, ik probeer alleen xbmc aan te spreken in de addon.py zodat ik de rest los kan aanroepen om te testen. Dat gaat een stuk fijner met debuggen